indra/llrender/llfontfreetype.h

```diff
@@ -43,15 +43,28 @@ typedef struct FT_FaceRec_* LLFT_Face;
 struct FT_StreamRec_;
 typedef struct FT_StreamRec_ LLFT_Stream;
 
+namespace nd
```

Comment:
Maybe change the namespace to "ll"? 😉

Reply:
Whoops 😬 
You're right, will do that tomorrow